triage/views.py

```diff
@@ -95,7 +96,46 @@ def dispatch(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        persona = forms.get_persona(self.triage_answers)
-        context['persona'] = persona
         context['triage_result'] = self.triage_answers
+        context['section_configuration'] = self.get_section_configuration()
         return context
+
+    def get_section_configuration(self):
+        persona = forms.get_persona(self.triage_answers)
```

Comment:
feels like `get_persona` should be in utils/helpers

Reply:
`get_persona` needs to know the shape of the data generated by the forme - and it makes sense to keep it in one place in case field name changes then we can grep a single file to make the changes.

Reply:
`we can grep a single file to make the changes` that argument can be used to argue for keeping all of the code in one file :-P

IMHO just forms and things used by them should be in `forms.py`, while it seems to not be the case here.